merlin/models/tf/core/encoder.py

```diff
@@ -159,11 +166,14 @@ def batch_predict(
 
         return merlin.io.Dataset(predictions)
 
-    def call(self, inputs, training=False, testing=False, targets=None, **kwargs):
+    def call(self, inputs, targets=None, training=False, testing=False, **kwargs):
```

Comment:
is the re-ordering of the params a functional change. e.g. are there any places where these are being passed in as positional params instead of keyword. One option could be to enforce this by adding a `*` after inputs. That way we can be sure that the order of the keyword arguments don't matter. 

```python
def call(self, inputs, *, targets=None, training=False, testing=False, **kwargs):
```

Reply:
Done